src/app/products/_components/ProductCard.tsx

```diff
@@ -0,0 +1,50 @@
+import Icon from '@/components/common/Icon';
+
+interface CardProps {
+  // imgUrl?: string
+  brandName: string; //브랜드명
+  productName: string; //제품명
+  price: string; //가격
+  discountPercentage?: string; //할인률
+}
+
+const ProductCard = ({
+  // imgUrl,
+  brandName,
+  productName,
+  price,
+  discountPercentage,
+}: CardProps) => {
+  const onClick = () => {};
+  return (
+    <div>
+      <div className="flex h-[27.1rem] w-[17.1rem] flex-col">
+        <div className="relative">
+          <img
+            className="top-0 left-0 rounded-xl"
+            src="https://placehold.co/171x171"
+          />
+          <button
+            type="button"
+            className="absolute top-[13.5rem] left-[13.5rem] z-10 text-gray-600"
```

Comment:
이번 제 브랜치 머지 되면 `z-index`를 변수로 쓰면 좋을 것 같아요 !
```suggestion
            className="absolute top-[13.5rem] left-[13.5rem] z-10 text-gray-600"
```
```suggestion
            className="absolute top-[13.5rem] left-[13.5rem] z-button text-gray-600"
```

`z-index`를 `z-modal`, `z-header`처럼 의미 있는 변수로 관리하면 나중에 유지보수에 더 좋을 것 같습니닷